addon/adapters/metaschema.js

```diff
@@ -1,3 +1,12 @@
 import OsfAdapter from './osf-adapter';
 
-export default OsfAdapter.extend({});
+export default OsfAdapter.extend({
+    buildURL(modelName, id, snapshot, requestType, query) {
+        let metaschemaType = '';
+        if (query) {
```

Comment:
This will only work for `.query()` or `.queryRecord()`. It would probably make more sense to move it to override [urlForQuery](https://emberjs.com/api/ember-data/3.1/classes/DS.BuildURLMixin/methods/urlForQuery?anchor=urlForQuery) instead of buildURL. At a minimum we should have a comment explaining what this is for/what queries it covers and a test (see [other nested route tests for example](https://github.com/CenterForOpenScience/ember-osf/pull/370/files#diff-5754255d4760774d76906689a995abf4)).

Abram has a [WIP PR up for nested routes](https://github.com/CenterForOpenScience/ember-osf-web/pull/197) but it doesn't cover `.query()`.

@alexschiller I'll let you make the final call on what you want here.